hash/x68k_flop.xml

```diff
@@ -27605,6 +27605,19 @@ No sound (verify)
 		</part>
 	</software>
 
+	<!-- boot OK -->
+	<software name="lforce" supported="yes">
+		<description>Last Force - The Virtual Battle in 1930's</description>
+		<year>1993</year>
+		<publisher>&lt;doujin&gt;</publisher>
+		<info name="developer" value="Hong Kong Project" />
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="1281968">
+				<rom name="last force - the virtual battle in 1930's (1993)(hong kong project).d88" size="1281968" crc="5df0a3d1" sha1="c2ef029af1b619d4fab9ca441f122e848aa5824a" />
```

Comment:
Is it okay to have that `'` in filename?

Reply:
128 other files in this software list already do.